src/server/scripts/EasternKingdoms/ScarletEnclave/chapter1.cpp

```diff
@@ -828,6 +789,64 @@ class npc_salanar_the_horseman : public CreatureScript
     }
 };
 
+class spell_deliver_stolen_horse : public SpellScriptLoader
+{
+public:
+    spell_deliver_stolen_horse() : SpellScriptLoader("spell_deliver_stolen_horse") { }
+
+    class spell_deliver_stolen_horse_SpellScript : public SpellScript
+    {
+        PrepareSpellScript(spell_deliver_stolen_horse_SpellScript);
+
+        bool Validate(SpellInfo const* /*spellInfo*/) override
+        {
+            return ValidateSpellInfo({ SPELL_DELIVER_STOLEN_HORSE, SPELL_EFFECT_STOLEN_HORSE });
+        }
+
+        void CheckHitSalanar(SpellEffIndex effIndex)
```

Comment:
```
src/server/scripts/EasternKingdoms/ScarletEnclave/chapter1.cpp:806:44:
 fatal error: unused parameter 'effIndex' [-Wunused-parameter]
        void CheckHitSalanar(SpellEffIndex effIndex)
                                           ^
1 error generated.
```
This parameter needs to be commented out:
```cpp
        void CheckHitSalanar(SpellEffIndex /*effIndex*/)
```

Reply:
All right, I'll comment it out when I'm back from work this evening!
Hope it's all 8)